### PR TITLE
Template rendering server-side

### PR DESCRIPTION
**Pull request by MichaelDurfey**

### CODE REVIEW

**Comment by MichaelDurfey** [comment]
Ticket: https://trello.com/c/VjhEBBem/18-set-up-proxy-server

File: proxy.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+app.use(morgan('dev'));
+app.set('view engine', 'pug')
+
+app.set('port', process.env.PORT || 3004);
+
+app.use(express.static(path.join(__dirname)));
+app.use(express.static(path.join(__dirname, '../title-map/client/dist')));
+
+// app.use('')
```
Comment (ZeMunchkin) [inline proxy.js]: Take out comments before the commit

File: proxy.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+app.get('/restaurant/:id', (req, res) => {
+  console.log(req.params.id)
```
Comment (ZeMunchkin) [inline proxy.js]: Take out the console.log

File: router.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    }).pipe(res)
+  })
+  .get('/information/:id', (req, res) => {
+      request({
+      url: `http://localhost:3400${req.path}`,
+      method: req.method,
+    }).pipe(res)
+  });
+
+  module.exports = router;
```
Comment (ZeMunchkin) [inline router.js]: new line

**Review by ZeMunchkin — COMMENTED** [review]
Minor linting changes before you merge

**Pull request merged**